src/components/PartnershipCard/PartnershipCard.graphql

```diff
@@ -10,9 +10,24 @@ fragment PartnershipCard on Partnership {
   mouStatus {
     value
   }
-  organization {
-    name {
-      value
+  partner {
```

Comment:
I think since this is the card component and `PartnershipItem` is for the summary scene, the two queries should be isolated.  They just happen to have the save values for now.